From 57075fdc3f99b8d0b51fef994622704592fc7eb7 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Wed, 26 Aug 2015 19:21:00 +0200
Subject: [PATCH 10/12] kbuild: Enable DT symbols when CONFIG_OF_OVERLAY is
 used

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
---
Replacement for "ARM: DT: Enable symbols when CONFIG_OF_OVERLAY is
used".
---
 scripts/Makefile.lib | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 3d599716940c..aed0dfff505e 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -315,6 +315,10 @@ cmd_dt_S_dtb=						\
 $(obj)/%.dtb.S: $(obj)/%.dtb FORCE
 	$(call if_changed,dt_S_dtb)
 
+ifeq ($(CONFIG_OF_OVERLAY),y)
+DTC_FLAGS += -@
+endif
+
 quiet_cmd_dtc = DTC     $@
 cmd_dtc = $(HOSTCC) -E $(dtc_cpp_flags) -x assembler-with-cpp -o $(dtc-tmp) $< ; \
 	$(DTC) -O $(patsubst .%,%,$(suffix $@)) -o $@ -b 0 \
-- 
2.28.0

